myadsp/templates/email.html

```diff
@@ -53,8 +53,20 @@
                                 <td align="center" valign="top" style="font-family:Arial;">
                                     <table border="0" cellpadding="0" cellspacing="0" width="100%" id="emailBody" style="">
                                         <tr>
-                                            <td align="center" valign="center" background="https://ui.adsabs.harvard.edu/styles/img/background.jpg" style="width:100%; background-color: #150E35; padding: 10px;" >
-                                                <p><img src="https://ui.adsabs.harvard.edu/styles/img/ads_logo.png" alt="Astrophysics Data System" style="width: 50%; color: #ffffff; font-size: 34px; font-family: Arial;"/></p>
+                                            <td align="center" valign="top" id="m_4393282051944905389m_-5269006104307297584templateHeader" style="background:#222222 url(&quot;https://ci3.googleusercontent.com/meips/ADKq_NbWD9eAlbuABElVbFQdmkeLIM77u3maXBPXOELnNNSunHZjJPYgSI3sNbwJe2DflvWFjSsuYrmBmf0c6t9EpPYo4SB_GMN_NbNVPj2-211hCOy4O0Xj__LxqnXB1gY2k6EOThxVlC1c2RaU3FIpIfyzcfXCaAO5l08=s0-d-e1-ft#https://mcusercontent.com/ed0cf26186d0cdd7bad11ed29/images/7d074991-6d2a-8279-c6bf-6150e8d7020c.png&quot;) no-repeat center/cover;background-color:#222222;background-image:url(https://ci3.googleusercontent.com/meips/ADKq_NbWD9eAlbuABElVbFQdmkeLIM77u3maXBPXOELnNNSunHZjJPYgSI3sNbwJe2DflvWFjSsuYrmBmf0c6t9EpPYo4SB_GMN_NbNVPj2-211hCOy4O0Xj__LxqnXB1gY2k6EOThxVlC1c2RaU3FIpIfyzcfXCaAO5l08=s0-d-e1-ft#https://mcusercontent.com/ed0cf26186d0cdd7bad11ed29/images/7d074991-6d2a-8279-c6bf-6150e8d7020c.png);background-repeat:no-repeat;background-position:center;background-size:cover;border-top:0;border-bottom:0;padding-top:153px;padding-bottom:153px">
```

Comment:
We should talk to the UI folks to see if we can host these images in the `styles/` directory as we did with the old ones.

Reply:
Asked Jennifer.

Reply:
Okay.